.changeset/grandma-bicycle-wheels.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/test-utils': patch
+---
+
+When providing `mountedRoutes` to any of `createTestAppWrapper()`, `wrapInTestApp()`, or `renderInTestApp()`, it's now possible to mount a `SubRouteRef`.
```

Comment:
Not sure about this tbh, you'd typically mount the route ref and any sub route refs would be relative to that right?

Reply:
`:mindblown:` I swear this was necessary to get a call to `useRouteRef(someSubRouteRef)` in a rendered component within a test to work.  ...But coming back to it now, I guess it doesn't?

Pushing up a version w/out the test-utils change.  Thanks @Rugvip.